node_modules/col-core/lib/db.js

```diff
@@ -242,6 +263,10 @@ var setUpModel = function(sequelize) {
       'type': Sequelize.STRING,
       'allowNull': true
     },
+    'canvas_email': {
```

Comment:
I missed that you can include the email address when requesting data from the API. I've added it to the poller. That way people who haven't launched into the LTI tool who are added to a whiteboard, will get an email when a chat message is added to it